bokehjs/src/lib/models/callbacks/open_url.ts

```diff
@@ -29,16 +29,19 @@ export class OpenURL extends Callback {
     }))
   }
 
+  raise(url: string): void {
```

Comment:
Maybe this is some kind of idiom I don't know, but I wouldn't associate this name with the following implementation, without seeing the implementation.

Reply:
I am using it in the sense of "raise a browser window" which is common enough at least  in my experience, but I'm happy to change it if you have any specific suggestion.

Reply:
I guess I recall this from some old WinAPI nomenclature, but still this doesn't sound right to me. Maybe `navigate` (which is web standard call this)?

Reply:
I went ahead and renamed it to `navigate` when fixing tests.